src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -149,15 +149,12 @@
 
         $(".dropdown-menu .dropdown-item").on("click", function () {
             if ($(":checkbox[name='itemIds']:checked").length > 1) {
-
-                var title = $(this).data('title');
-                var message = $(this).data('message');
-                confirmDialog(title, message, r => {
+                confirmDialog({title: $(this).data('title'), message: $(this).data('message'), callback: r => {
```

Comment:
arrow function are not supported in IE https://caniuse.com/#feat=arrow-functions